src/tween.ts

```diff
@@ -1,125 +1,90 @@
-import { engine, Entity, IEngine, TransformType, Schemas, Transform, EntityState } from '@dcl/sdk/ecs'
-import { Scalar, Vector3, Quaternion } from '@dcl/sdk/math'
-import { InterpolationType, interpolate } from './math'
+import { engine, Entity, IEngine, EntityState, Tween, EasingFunction, TweenHelper } from '@dcl/sdk/ecs'
+
 import { priority } from './priority'
+import { Scalar } from '@dcl/sdk/math'
 
 export type OnFinishCallback = () => void
-
 export type Tweens = ReturnType<typeof createTweens>
+type TweenMap = Map<Entity, {
+  normalizedTime: number
+  callback: OnFinishCallback | undefined
+}>
 
-function createTweens(targetEngine: IEngine) {
-  const Vector3TweenSchema = {
-    start: Schemas.Vector3,
-    end: Schemas.Vector3,
-    interpolationType: Schemas.EnumString(InterpolationType, InterpolationType.LINEAR),
-    speed: Schemas.Float,
-    normalizedTime: Schemas.Float
-  }
-  const PositionTween = targetEngine.defineComponent('dcl.utils.PositionTween', Vector3TweenSchema)
-  const ScaleTween = targetEngine.defineComponent('dcl.utils.ScaleTween', Vector3TweenSchema)
-  const RotationTween = targetEngine.defineComponent('dcl.utils.RotationTween', {
-    start: Schemas.Quaternion,
-    end: Schemas.Quaternion,
-    interpolationType: Schemas.EnumString(InterpolationType, InterpolationType.LINEAR),
-    speed: Schemas.Float,
-    normalizedTime: Schemas.Float
-  })
-
-  type FinishCallbackMap = Map<Entity, OnFinishCallback | undefined>
-
-  const positionFinishCbs: FinishCallbackMap = new Map()
-  const rotationFinishCbs: FinishCallbackMap = new Map()
-  const scaleFinishCbs: FinishCallbackMap = new Map()
+function createTweens(engine: IEngine) {
```

Comment:
this name is clashing with the local engine, I would rather keep the name as `targetEngine`